students/Ruohan/lesson01/break_me.py

```diff
@@ -1,22 +1,25 @@
-# NameError:
-#print ('NameError: ')
-#print (y)
+#NameError:
+def give_NameError():
+    print ('NameError: ')
+    print (y)
 
 #TypeError
-#print ('TypeError: ')
-#x = 2
-#y = 'hello'
-#print(x+y)
+def give_TypeError():
+....print ('TypeError: ')
```

Comment:
Do I see actual dots here at the beginning of this line? Wonder if it didn't give a SyntaxError in the first place?